packages/ui/src/components/DefaultMessage/styles.module.css

```diff
@@ -6,4 +6,8 @@
         width: 100%;
         height: 100%;
     }
+    &.empty-message {
+        padding-top: 40px;
+        height: 200px;
+    }
```

Comment:
This styling applied for all cases.
Use design tokens.
We need a padding of 40px between the main text and description